Xamarin.Forms.Platform.Android/FastRenderers/ImageElementManager.cs

```diff
@@ -146,6 +146,7 @@ async static Task TryUpdateBitmap(IImageRendererController rendererController, I
 			catch (Exception ex)
 			{
 				Log.Warning(nameof(ImageElementManager), "Error loading image: {0}", ex);
+				await Control.SetImagePlaceholder(newImage.ErrorSource).ConfigureAwait(false);
```

Comment:
I think there's a version of this that needs to be adding to UWP and iOS as well

UWP and iOS both have similar places where this could be set

Reply:
@PureWeen Maybe it's better to remove it. Because the way that `ImageElementManager` works on Android is different from iOS and UWP. On Android, we set the Image using an [external method ](https://github.com/xamarin/Xamarin.Forms/blob/4a2b823045a48b704d7cea561f3ed59bdf9f5c0f/Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs#L24) and we have an isolated [method to set the placeholder](https://github.com/xamarin/Xamarin.Forms/blob/4a2b823045a48b704d7cea561f3ed59bdf9f5c0f/Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs#L110), this doesn't occur on UWP and iOS.

For UWP, if the `ImageSource` doesn't exist or something went wrong the value will be `null` if that is the case, we will set the `ErrorSource`, as you can[ see here](https://github.com/xamarin/Xamarin.Forms/pull/11682/files#diff-8d87756798298b7c41f0e38383d8b106R160)
For iOS is similar also, as you can see [here](https://github.com/xamarin/Xamarin.Forms/pull/11682/files#diff-e7e02ce38379deb0c027bd4e04e17265R239) 

If you have any thoughts about it, please let me know.

Reply:
Can we move this to here?

https://github.com/xamarin/Xamarin.Forms/blob/4a2b823045a48b704d7cea561f3ed59bdf9f5c0f/Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs#L98

And if the source isn't null and nothing got set then set it to the error source?

Reply:
> And if the source isn't null and nothing got set then set it to the error source?

@PureWeen Are you saying at the end of the method? Something like this...

```csharp

//line 96 Xamarin.Forms.Platform.Android.ImageViewExtensions method
      finally
			{
				// only mark as finished if we are still working on the same image
				if (SourceIsNotChanged(newView, newImageSource))
				{
					imageController?.SetIsLoading(false);
					imageController?.NativeSizeChanged();
				}
                 // Add this at the end to see if we have an image in loaded
				if (imageView.Drawable != null)
				{
					await imageView.SetImagePlaceholder(newView?.ErrorSource);
				}
			}
```